docs/settings/alert-action-settings.asciidoc

```diff
@@ -269,15 +269,76 @@ A configuration URL that varies by connector:
 NOTE: If you are using the `xpack.actions.allowedHosts` setting, make sure the hostname in the URL is added to the allowed hosts.
 --
 
+`xpack.actions.preconfigured.<connector-id>.config.createCommentJson`::
```

Comment:
Cool that we're doc'ing the pre-configured connectors.  These should actually be exactly the same as the `config` and `secrets` parts of the connector types.  We just copy the values from the Kibana settings into our in-memory version of the connector config/secrets: https://github.com/elastic/kibana/blob/87c60a381936314db9474921b3d1da033f276ae7/x-pack/plugins/actions/server/plugin.ts#L252-L270

There's some special handling for the Alert History Index connector, which appears to be the only special thing there.

So it seems a shame to have to duplicate this.  I wonder if we could point to where these are already documented, or at least find some way long term of sharing these: https://www.elastic.co/guide/en/kibana/current/create-connector-api.html#create-connector-api-request-body 

If we do end up leaving this page as-is-ish, could we group the config keys by connector type?  Presumably someone coming here will want to know all the config / secrets for a specific type, so having them grouped seems like it would make sense.  Also seems like it will end up cluttering our already busy config settings page, once we start adding more.  Feels like a separate page for these could be better.

Reply:
Yes, I would love it if the settings could also be automatically generated! AFAIK no one is working on that, however, so for now rather than point people to multiple places I've just made them match.

I grouped the examples connector type in the preconfigured connectors page (https://www.elastic.co/guide/en/kibana/master/pre-configured-connectors.html#_examples_3), but since those examples don't cover every option for all the connectors, I'm treating the settings page as the definitive source of truth. Once I get all the preconfigured connector examples moved, I'll add a note to that effect and a link to the settings page. 

For now, in the hope that the setting will somehow/someday be automatically generated, I won't try to split it out and will leave as alphabetical sorting same as other settings pages unless it becomes too problematic.